packages/neo-one-node-vm/src/__tests__/syscalls.test.js

```diff
@@ -98,19 +98,21 @@ const asset = {
   available: new BN(5),
 };
 
-// const sbContract = new ScriptBuilder();
-// sbContract.emitSysCall('Neo.Contract.GetStorageContext');
-// const contract = new Contract({
-//   script: sbContract.build(),
-//   parameterList: [],
-//   returnType: CONTRACT_PARAMETER_TYPE.VOID,
-//   name: '',
-//   codeVersion: '',
-//   author: '',
-//   email: '',
-//   description: '',
-//   contractProperties: CONTRACT_PROPERTY_STATE.HAS_STORAGE,
-// })
+const iterator = AsyncIterableX.of();
+const nextItem = {
+  value: new StorageItem({
+    hash: scriptAttributeHash,
+    key: Buffer.from('key', 'utf-8'),
+    value: Buffer.from('val', 'utf-8'),
+  }),
+};
+// $FlowFixMe
+iterator.next = jest.fn(() => Promise.resolve(nextItem));
+const stackIterator = new StackItemIterator((iterator: $FlowFixMe));
+async function initializeIterator() {
+  await stackIterator.next();
+}
+initializeIterator();
```

Comment:
This causes a race condition with the tests since we're not waiting for it to complete. It's possible the test runs before `initializeIterator` is actually called. Add a key to `SysCall` called like `pretest` that is an optional async function to execute before the test executes.